<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Store</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/Music.svg</normaloff>:/Music.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1100</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: #ffffff;
}</string>
    </property>
    <widget class="QWidget" name="user_page">
     <widget class="QPushButton" name="pushButton_logout">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>39</y>
        <width>75</width>
        <height>75</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fafafa;
	border: 0px;
	border-radius: 37px; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/navigate_next.svg</normaloff>:/navigate_next.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_clientname">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>40</y>
        <width>473</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #1d1b20;
	background-color: transparent;
	font: 700 21pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>96</y>
        <width>671</width>
        <height>463</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea{
	background-color: transparent;
	border: 0px;
}

QScrollBar:vertical {
        border: none;
        background: transparent;
        width: 10px;
        margin: 0px 0px 0px 0px;
    }
    QScrollBar::handle:vertical {
        background: #9b9c9c;
        min-height: 20px;
        border-radius: 5px;
    }
    QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
        background: none;
        height: 0px;
    }
    QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
        background: transparent;
    }</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>981</width>
         <height>685</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="login">
     <widget class="QLineEdit" name="lineEdit_login">
      <property name="geometry">
       <rect>
        <x>345</x>
        <y>211</y>
        <width>409</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	color: #1d1b20;
	background-color: #fafafa;
	border-radius: 13px; 
	font: 16pt &quot;JetBrains Mono&quot;;
	padding-left: 27px;
}

QLineEdit:focus {
    border: 2px solid #140f10;  
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>345</x>
        <y>255</y>
        <width>409</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	color: #1d1b20;
	background-color: #fafafa;
	border-radius: 13px; 
	font: 16pt &quot;JetBrains Mono&quot;;
	padding-left: 27px;
}

QLineEdit:focus {
    border: 2px solid #140f10;  
}</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhHiddenText|Qt::InputMethodHint::ImhNoAutoUppercase|Qt::InputMethodHint::ImhNoPredictiveText|Qt::InputMethodHint::ImhSensitiveData</set>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>506</x>
        <y>310</y>
        <width>89</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fafafa;
	color: #140f10;
	font: 16pt &quot;JetBrains Mono&quot;;
	border-radius: 12px;
}</string>
      </property>
      <property name="text">
       <string>Вход</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>0</y>
        <width>671</width>
        <height>560</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea{
	background-color: transparent;
	border: 0px;
}

QScrollBar:vertical {
        border: none;
        background: transparent;
        width: 10px;
        margin: 0px 0px 0px 0px;
    }
    QScrollBar::handle:vertical {
        background: #9b9c9c;
        min-height: 20px;
        border-radius: 5px;
    }
    QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
        background: none;
        height: 0px;
    }
    QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
        background: transparent;
    }</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>981</width>
         <height>685</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="cart">
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>96</y>
        <width>671</width>
        <height>464</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea{
	background-color: transparent;
	border: 0px;
}

QScrollBar:vertical {
        border: none;
        background: transparent;
        width: 10px;
        margin: 0px 0px 0px 0px;
    }
    QScrollBar::handle:vertical {
        background: #9b9c9c;
        min-height: 20px;
        border-radius: 5px;
    }
    QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
        background: none;
        height: 0px;
    }
    QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
        background: transparent;
    }</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>981</width>
         <height>685</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_cart_total">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>40</y>
        <width>473</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #1d1b20;
	background-color: transparent;
	font: 700 21pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clean_cart">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>19</y>
        <width>75</width>
        <height>75</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fafafa;
	border: 0px;
	border-radius: 37px; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/delete.svg</normaloff>:/delete.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_submit_cart">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>469</y>
        <width>75</width>
        <height>75</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fafafa;
	border: 0px;
	border-radius: 37px; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/check.svg</normaloff>:/check.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_cart_total_2">
      <property name="geometry">
       <rect>
        <x>392</x>
        <y>71</y>
        <width>315</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #9b9c9c;
	background-color: transparent;
	font: 12pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
